artifacts/centosstream/centos-stream.go

```diff
@@ -29,10 +29,12 @@ type centos struct {
 
 func (c *centos) Metadata() *api.Metadata {
 	return &api.Metadata{
-		Name:                   "centos-stream",
-		Version:                c.Version,
-		Description:            description,
-		ExampleUserDataPayload: c.UserData(&docs.UserData{}),
+		Name:        "centos-stream",
+		Version:     c.Version,
+		Description: description,
+		ExampleUserData: docs.UserData{
+			Username: "centos",
```

Comment:
The default user is created by cloud-init, therefore you cannot find it in the image.

It is called differently for `centos-stream:9' however.

```
$ virt-rescue -a ~/Downloads/CentOS-Stream-GenericCloud-9-20230417.0.x86_64.qcow2
[..]
><rescue> lsblk
NAME   MAJ:MIN RM  SIZE RO TYPE MOUNTPOINTS
sda      8:0    0   10G  0 disk 
`-sda1   8:1    0  7.8G  0 part 
sdb      8:16   0    4G  0 disk /
><rescue> mount /dev/sda1 /sysroot
><rescue> chroot /sysroot
><rescue> cat etc/redhat-release 
CentOS Stream release 9
><rescue> cat /etc/cloud/cloud.cfg
[..]
# System and/or distro specific settings
# (not accessible to handlers/transforms)
system_info:
   # This will affect which distro class gets used
   distro: centos
   # Default user name + that default users groups (if added/used)
   default_user:
     name: cloud-user
     lock_passwd: true
     gecos: Cloud User
     groups: [adm, systemd-journal]
     sudo: ["ALL=(ALL) NOPASSWD:ALL"]
     shell: /bin/bash
   # Other config here will be given to the distro class and/or path classes
   paths:
      cloud_dir: /var/lib/cloud/
      templates_dir: /etc/cloud/templates/
   ssh_svcname: sshd
```

In `centos-stream:9` the user is called `cloud-user`.